### DISCUSSION TITLE
[Bug] Move the Constraint markup extension to System.Maui.Xaml

### DISCUSSION THREAD

**Discussion by davidbritch** [dotnet/maui#205] [General]
### Description

It'd be good if [ConstraintExpression](https://github.com/dotnet/maui/blob/build/System.Maui.Core/ConstraintExpression.cs) gets moved from System.Maui.Core to the MarkupExtensions folder in the System.Maui.Xaml folder, there to live happily ever after with all the other markup extensions.

**Comment by PureWeen** [comment]
@StephaneDelcroix  ?

**Reply by StephaneDelcroix** [reply]
y, move it. do that in Forms for now
